src/shared/hooks/useDeleteResource.js

```diff
@@ -177,11 +173,20 @@ export function useDeleteResource({
     resourceTitle,
     resourceIsCluster = false,
     resourceUrl,
+    closeFn,
     deleteFn,
     cancelFn,
     additionalDeleteInfo,
     disableDeleteButton = false,
+    allowForceDelete = false,
```

Comment:
In my opinion `allowForceDelete` is not the best way to do this, it's used only to adjust the text in the Delete Button, so maybe change it to `customDeleteText` and pass the string here